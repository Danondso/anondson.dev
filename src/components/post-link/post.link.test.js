import React from 'react';
import renderer from 'react-test-renderer';
import PostLink from './post-link';

describe('External Link Snapshot test', () => {
  it('renders correctly', () => {
    const postData = {
      frontmatter: {
        path: '/some/path',
        title: 'Fake Title',
      },
    };

    const tree = renderer
      .create(<PostLink backgroundColor="var(--background-green)" key={1235} post={postData} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
