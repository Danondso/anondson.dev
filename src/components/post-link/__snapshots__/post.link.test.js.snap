// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`External Link Snapshot test renders correctly 1`] = `
<section
  alt="A default setup for a colored background section."
  style={
    Object {
      "backgroundColor": "#8FA998",
      "boxShadow": "0 3px 2px rgba(0, 0, 0, 0.15)",
      "padding": "4rem",
    }
  }
>
  <div
    style={
      Object {
        "color": "#3E4E50",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <a
      href="/some/path"
      style={
        Object {
          "color": "#FFF",
        }
      }
    >
      <h1>
        Fake Title
      </h1>
    </a>
  </div>
</section>
`;
